pkg/providerConfigs/adjustDriver.go

```diff
@@ -143,6 +145,16 @@ func AdjustGitopsRepo(
 		removeAllWithLogger(strings.ToLower(fmt.Sprintf("%s/%s-%s/templates/workload-vcluster/40-cloudflare-origin-issuer-crd.yaml", gitopsRepoDir, cloudProvider, gitProvider)))
 		removeAllWithLogger(strings.ToLower(fmt.Sprintf("%s/%s-%s/templates/workload-vcluster/41-cloudflare-origin-ca-issuer.yaml", gitopsRepoDir, cloudProvider, gitProvider)))
 		removeAllWithLogger(strings.ToLower(fmt.Sprintf("%s/%s-%s/templates/workload-vcluster/45-cloudflare-origin-issuer.yaml", gitopsRepoDir, cloudProvider, gitProvider)))
+
+		for _, cloud := range aiSupport {
+			if cloud == cloudProvider {
+				removeAllWithLogger(filepath.Join(gitopsRepoDir, strings.ToLower(cloudProvider), strings.ToLower(fmt.Sprintf("%s-%s", cloudProvider, gitProvider)), "templates", "ai-cluster", "cloudflare-origin-issuer"))
+				removeAllWithLogger(filepath.Join(gitopsRepoDir, strings.ToLower(cloudProvider), strings.ToLower(fmt.Sprintf("%s-%s", cloudProvider, gitProvider)), "templates", "ai-cluster", "40-cloudflare-origin-issuer-crd.yaml"))
+				removeAllWithLogger(filepath.Join(gitopsRepoDir, strings.ToLower(cloudProvider), strings.ToLower(fmt.Sprintf("%s-%s", cloudProvider, gitProvider)), "templates", "ai-cluster", "41-cloudflare-origin-ca-issuer.yaml"))
+				removeAllWithLogger(filepath.Join(gitopsRepoDir, strings.ToLower(cloudProvider), strings.ToLower(fmt.Sprintf("%s-%s", cloudProvider, gitProvider)), "templates", "ai-cluster", "45-cloudflare-origin-issuer.yaml"))
+				break
```

Comment:
Nitpick but to prevent having to `strings.ToLower` everything, if you're always thinking that the whole string is lowercase, you can just generate the path then wrap that into a `strings.ToLower()`, a path separator, `/` cannot be lowercased 😆

Reply:
xD :)